joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/NodeBuilders.scala

```diff
@@ -107,7 +107,7 @@ object NodeBuilders {
       .code(code)
       .signature("")
       .dispatchType(DispatchTypes.STATIC_DISPATCH)
-      .typeFullName(typeFullName.getOrElse(PropertyDefaults.TypeFullName))
+      .typeFullName(typeFullName.getOrElse("ANY"))
```

Comment:
Is there no generic constant for this? (It's used across most of our frontends, so there probably should be. But this doesn't have to block this PR.)

Reply:
Surprisingly no (at least not that I could find from a quick search). I agree it would be a good idea to have a single constant for this though.